docs/getting-started/migrate/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Migrate from the legacy Shopify App CLI
```

Comment:
I wonder if we can create a consistent name for the old version. I'm worried that we will eventually have other "old" versions, and it might be useful to have a more specific name for the git-repo version. I noticed the title here says "legacy" but in other places we just say "old".

"Original"? "original version installed with git"?

Reply:
The clearest way for someone to determine if they're using the "old"/"original" version is to run `shopify version` - the current ("old", "original", etc.) version doesn't support the `version` command!  The one we're building does.

Reply:
I'm going with `legacy` for now, to designate specifically the non-versioned, git-installed instance of the CLI. So we preserve "old/older" for any version that's not `@latest`